### PR TITLE
Parameter tiv

### PR DESCRIPTION
**Pull request by JulietteFrancovich**
Added tidal impedance variation parameter computation for global impedance and pixel impedance. Also added to the breath detection algorithm to allow breath detection on a pixel level.

### CODE REVIEW

File: eitprocessing/features/breath_detection.py [hunk 1]
```diff
@@ -295,7 +295,7 @@ def _remove_breaths_around_invalid_data(
 
         return breaths
 
-    def find_breaths(self, data: np.ndarray) -> list[Breath]:
+    def find_breaths_1d(self, data: np.ndarray) -> list[Breath]:
```
Comment (psomhorst) [inline eitprocessing/features/breath_detection.py:298]: Please don't rename functions that are used by other parts of the code or existing projects. This breaks all code using this feature.

Breaking code functionality should be done sparingly. In this case, I don't think it is necessary. 

Instead, if it is necessary to distinguish between data with different dimensionality, build a check into this function that is backwards compatible, but ensures it also handles new cases.

File: eitprocessing/features/breath_detection.py [hunk 2]
```diff
@@ -355,3 +355,62 @@ def find_breaths(self, data: np.ndarray) -> list[Breath]:
         ]
 
         return self._remove_breaths_around_invalid_data(breaths, data)
+
+
+    def find_breaths_2d(self, data: np.ndarray) -> np.ndarray:
```
Comment (psomhorst) [inline eitprocessing/features/breath_detection.py:360]: It seems this code is mostly a copy of another function. If this function applies an algorithm implemented in an existing function several times, why not call that function several times?

File: eitprocessing/parameters/tidal_impedance_variation.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        result = {}
+        for name, function in self.summary_stats_global.items():
+            result[name] = function(tiv_values)
+
+        result["peak indices"] = middle_indices
+
+        return result
+
+
+    def compute_pixel_parameter(self, sequence: Sequence, data_label: str, tiv_method = 'inspiratory') -> dict | list[dict]:
```
Comment (psomhorst) [inline eitprocessing/parameters/tidal_impedance_variation.py:89]: It seems to me that this function is for a large part a copy of `compute_global_parameter()`. Is it possible to extract the part(s) that are duplicated into separate functions that are used by these functions?

**Review by psomhorst — COMMENTED** [review]
I have looked at the structure of the code, but not at the functionality or the code details. Could you address my concerns before I continue looking at the functionality and details?

**Comment by JulietteFrancovich** [comment]
@psomhorst Thank you for your comments, I have addressed them and pushed new commits.

**Pull request closed without merging**